client/src/api/api.js

```diff
@@ -284,10 +303,28 @@ export const deleteAudioFile = async (courseId, unitId, lessonId, vocabId) => {
   if (!body.success || body.success === 'false') {
     throw new Error(body.message)
   }
+  await deleteFileURI(vocabId, 'audio')
   return body
 }
 
 /* Image Endpoints */
+export const uploadImageFileToExpo = async (vocabId, uri) => {
```

Comment:
Since this is basically the image version of `downloadAudioFileToExpo`, I would try to make the function names similar